documentation/docs/03_account.md

```diff
@@ -9,209 +9,233 @@ It is responsible for the creation and management of account addresses and their
+| --------------------- | --------- | --------------------------------- | ----------------------------------------------------------------- |
+| `address`             | ✓         | The latest address of the account | "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3" |
+| `immutable_metadata`  | ✓         | None                              | "{ key: value }"                                                  |
+| `metadata`            | ✓         | None                              | "{ key: value }"                                                  |
 
-#### Example(s)
+#### Examples
 
+Mint a NFT to the latest address of the account.
 ```sh
 > Account "main": mint-nft
 ```
 
-### `new-address`
+Mint a NFT to a given address.
+```sh
+> Account "main": mint-nft "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3"
+```
 
-#### Parameters
+Mint a NFT to a given address with immutable metadata and metadata.
+```sh
+> Account "main": mint-nft "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3" "{ key: value }" "{ key: value }"
+```
 
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+### `new-address`
 
-#### Example(s)
+Generates a new address.
+
+#### Example
 
 ```sh
 > Account "main": new-address
 ```
 
 ### `send`
 
+Sends an amount to an address.
+
 #### Parameters
 
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+| Name      | Optional  | Default | Example                                                           |
+| --------- | --------- | ------- | ----------------------------------------------------------------- |
+| `address` | ✘         | N/A     | "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3" |
+| `amount`  | ✘         | N/A     | 1000000                                                           |
 
-#### Example(s)
+#### Example
 
 ```sh
-> Account "main": send
+> Account "main": send rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 1000000
 ```
 
 ### `send-micro`
 
+Sends a micro amount to an address.
+
 #### Parameters
 
-| Name    | Optional  | Default       | Example |
-| ------- | --------- | ------------- | ------- |
-| `` | ✓ | | |
+| Name      | Optional  | Default | Example                                                           |
+| --------- | --------- | ------- | ----------------------------------------------------------------- |
+| `address` | ✘         | N/A     | "rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3" |
+| `amount`  | ✘         | N/A     | 1                                                                 |
 
-#### Example(s)
+#### Example
 
 ```sh
-> Account "main": send-micro
+> Account "main": send-micro rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 1
 ```
 
 ### `send-native-token`
 
+Sends native tokens to an address.
```

Comment:
Here also maybe with `with StorageDepositReturn and Expiration Unlock Conditions`?